installer/InstallBuilder/linuxrpm.py

```diff
@@ -146,9 +147,9 @@ def StageAndProperlyNameRPM(self):
                 self.fullversion_dashed + '.rhel.' + \
                 str(self.variables['PFMAJOR'])  + '.' + self.variables['PFARCH'] + '.rpm'
         elif self.variables['PFDISTRO'] == 'ULINUX':
-            rpmNewFileName = self.variables['SHORT_NAME'] + '-' + \
-                self.fullversion_dashed + '.universalr.' + \
-                str(self.variables['PFMAJOR'])  + '.' + self.variables['PFARCH'] + '.rpm'
+			rpmNewFileName = self.variables['SHORT_NAME'] + '-' + \
+				self.fullversion_dashed + '.universalr.' + \
+				str(self.variables['PFMAJOR'])  + '.' + self.variables['PFARCH'] + '.rpm'
```

Comment:
So why did you change from space indenting to tab indenting here?

Is there an actual change here at all? If not, please restore indenting and eliminate this change entirely, I think it's just unnecessary churn.